juniper_official/Solutions/Microburst/microburst.rule

```diff
@@ -86,6 +86,26 @@ iceberg {
                 type string;
                 description "Queue number or name to monitor, regular expression, eg '1|2'";
             }
+            rule-properties {
+                version 2;
+                contributor juniper;
+                supported-healthbot-version 1.0.1;
+                catalogue {
+                    tier 1;
+                }
+                supported-devices {
+                    juniper {
+                        operating-system junos {
+                            products MX {
+                                releases 17.2 {
+                                    release-support min-supported-release;
+                                    platform All-i386-with-FPC7-and-above;
```

Comment:
"All-i386" is not a platform entry.. Please have a valid list of platform name. Example MX120, MX2020. This will be used in the future releases and matched against the device-facts

For FPC Related facts, Please use the apply-macro